aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/internal/S3V4AuthErrorRetryStrategy.java

```diff
@@ -107,7 +107,9 @@ private AuthRetryParameters redirectToS3External() {
     }
 
     private AWSS3V4Signer buildSigV4Signer(final String region) {
-        AWSS3V4Signer v4Signer = new AWSS3V4Signer();
+        // Retry with chunkedEncoding set to true.
+        boolean isChunkedEncoding = true;
+        AWSS3V4Signer v4Signer = new AWSS3V4Signer(isChunkedEncoding);
```

Comment:
For your use case this class will never be used so this is fine but it may be confusing if another developer sets the configuration option and it isn't honored in the retry policy.